python/py-pyaudio/Portfile

```diff
@@ -22,18 +23,17 @@ homepage                http://people.csail.mit.edu/hubert/pyaudio/
 master_sites            pypi:P/PyAudio/
 
 distname                ${my_name}-${version}
-
 checksums               md5     7e4c88139284033f67b4336c74eda3b8 \
                         rmd160  7a6bb88f56622555e77eb799e4ee74ff970b6e92 \
                         sha256  93bfde30e0b64e63a46f2fd77e85c41fd51182a4a3413d9edfaf9ffaa26efb74
 
 worksrcdir              PyAudio-${version}
 
-python.versions         27 34 35 36
+python.versions         27 34 35 36 37
 
 if {${name} ne ${subport}} {
+    depends_build       port:py${python.version}-setuptools
     depends_build-append \
-                        port:py${python.version}-setuptools \
```

Comment:
What was the purpose of moving this and creating an unneeded `depends_build` line? It seems arbitrary, and unnecessary for the change you made. Unless you have a good reason to have done this, please revert it.